types/meteor/globals/reactive-dict.d.ts

```diff
@@ -1,5 +1,5 @@
 
-declare class ReactiveDict<O = EJSONable> {
+declare class ReactiveDict<O = { [key: string]: EJSONable }> {
```

Comment:
1. As mentioned above, this should be a separate PR.
2. How about `Record<string, EJSONable>`?